### ISSUE TITLE
Not receiving response back from Twiml

### ISSUE THREAD

**Issue by ueg1990** [twilio/twilio-python#213]
Hello, 

I am trying to get a response back from Twiml after sending a text to my twilio number. When I send the text, it is sent successfully (i see a 200 OK) but i do not receive a response. I am using Python 3.4, flask-restful 0.3.2, twilio 4.4.0 and here is the code i have:

from flask import request
from flask.ext import restful
from twilio import twiml

class SMS(restful.Resource):

    def post(self):
        response = twiml.Response()
        user_input = request.form['Body']
        user_from = request.form['From']
        print(request.form)
        msg = "success!!!"
        response.message(msg)
        return str(response)

Any help would be greatly appreciated it.

**Comment by ueg1990** [comment]
I found a solution but i wanted to know if this is a new change or something? The solution required using Response object from class to change the header to 'text/xml':

from flask import request, Response
from flask.ext import restful
from twilio import twiml

class SMS(restful.Resource):

    def post(self):
        response = twiml.Response()
        user_input = request.form['Body']
        user_from = request.form['From']
        print(request.form)
        msg = "success!!!"
        response.sms(msg)
        return Response(str(response), content_type='text/xml')

In my previous project (this was around January 2015), I did not need to return a response object with header 'text/xml'. Before this issue is closed, does anyone have an idea why there is a difference now? 

Thank you :)
